run_linux.go

```diff
@@ -543,6 +543,33 @@ rootless=%d
 
 	defer b.cleanupTempVolumes()
 
+	// Handle mount flags that request that the source locations for "bind" mountpoints be
+	// relabeled, and filter those flags out of the list of mount options we pass to the
+	// runtime.
+	for i := range spec.Mounts {
+		switch spec.Mounts[i].Type {
+		default:
+			continue
+		case "bind", "rbind":
+			// all good, keep going
+		}
```

Comment:
```suggestion
		if switch spec.Mounts[i].Type == "bind" ||  switch spec.Mounts[i].Type  == "rbind" {
		
```

Do you think `if` is easier for readability ? We are just handling one case.